models/ios/prefix_lists/ios_prefix_lists.yaml

```
@@ -0,0 +1,86 @@
+module: ios_prefix_lists
+short_description: Prefix Lists resource module
+description:
+  - This module configures and manages the attributes of prefix list on Cisco IOS.
+version_added: 2.2.0
+author: Sumit Jaiswal (@justjais)
+notes:
+  - Tested against Cisco IOSv Version 15.2 on VIRL
+  - This module works with connection C(network_cli).
+    See L(IOS Platform Options,../network/user_guide/platform_ios.html)
+options:
+  config:
+    description: A list of configurations for Prefix lists.
+    type: list
+    elements: dict
+    suboptions:
+      afi:
+        description:
+          - The Address Family Indicator (AFI) for the  prefix list.
+        type: str
+        choices: ['ipv4', 'ipv6']
+      prefix_lists:
+        description: List of Prefix-lists.
+        type: list
+        elements: dict
+        suboptions:
+          name:
+            description: Name of a prefix-list
+            type: str
+          params:
+            description: Prefix-lists supported params.
+            type: list
+            elements: dict
+            suboptions:
+              action:
+                description: Specify packets to be rejected or forwarded
+                type: str
+                choices: ['deny', 'permit']
+              sequence:
+                description: sequence number of an entry
+                type: int
+              description:
+                description:  Prefix-list specific description
+                type: str
+              address:
```

Since this expected input is in CIDR format.
```suggestion
              prefix:
```